ext/reflection/tests/ReflectionProperty_isFinal.phpt

```diff
@@ -12,6 +12,8 @@ class C {
     public protected(set) final mixed $p6;
     public private(set) mixed $p7;
     public private(set) final mixed $p8;
+
+    public function __construct( final $p9 ) {}
```

Comment:
```
    public function __construct( $p9 ) {}
```

It would be good to test also the case without the final property in the constructor.